fe/fe-core/src/main/java/org/apache/doris/mtmv/MTMVService.java

```diff
@@ -131,11 +133,11 @@ public void dropTable(Table table) {
         }
     }
 
-    public void alterTable(Table table, String oldTableName) {
-        Objects.requireNonNull(table);
-        LOG.info("alterTable, tableName: {}", table.getName());
+    public void alterTable(BaseTableInfo oldTableInfo, Optional<BaseTableInfo> newTableInfo, boolean isReplace) {
+        Objects.requireNonNull(oldTableInfo);
+        LOG.info("alterTable, tableName: {}", oldTableInfo);
```

Comment:
if u want to log info ,please let the log message more clearly, current message is not good because we could not related it with mtmv

Reply:
In the current method, we cannot determine which MTMVs are affected. This log is printed primarily to prove that the table modification has indeed entered the MTMV logic.